### DISCUSSION TITLE
1.79.1.yr0-ROCm customtkinter python module required! error

### DISCUSSION THREAD

**Discussion by ccbadd** [YellowRoseCx/koboldcpp-rocm#85] [Q&A]
Just tried to compile 1.79.1.yr0-ROCm and I get this error when compile is done:
````
Welcome to KoboldCpp - Version 1.79.1.yr0-ROCm
For command line arguments, please refer to --help
***
Warning, GUI failed to start: Reason: No module named 'customtkinter'
File selection GUI unsupported.
customtkinter python module required!
Please check command line: script.py --help
````
I am running Fedora 41 KDE and this is the first time I have tried to run koboldcpp on this machine.  Any ideas what is missing?

**Comment by ccbadd** [comment]
I went ahead and ran the koboldcpp.py script and it did compile and run the non rocm version.  After I went ahead and tried again and have the same issue still.

**Reply by R4di04kt1v3** [reply]
Hello, try this:

pip install ntk

A few weeks ago, I had the same problem, and for me, this was the fix.

**Reply by ccbadd** [reply]
> pip install ntk

just tried it and I get the same issue.  Thanks for the info though.
